### PR TITLE
Bug fixes and more features.

### PR DESCRIPTION
**Pull request by igorescobar**
Hi @oesmith!

A little bug fixes and now, ralio is able to create and delete story tasks. 
Hope you enjoy it.
- Igor

### CODE REVIEW

File: lib/ralio.js [hunk 1]
```diff
@@ -340,6 +347,106 @@ Ralio.prototype.point = function (storyID, points, callback) {
...
+        url: this.bulkUrl({"pathname":"/slm/webservice/1.36/task.js?query=(FormattedID = " + taskID + ")"}),
+        method: 'GET',
+        json: {},
+        strictSSL: !Ralio.test
+      };
+
+  this.request(options, function(error, data){
+    resource = data.QueryResult.TotalResultCount === 0 ? 
+                  "Task Not Found!".red : data.QueryResult.Results[0];
+    callback(resource);
```
Comment (oesmith) [inline lib/ralio.js]: Could you perhaps use a standard (error, resource) pair of arguments for the callbacks here? And handle error cases where you're calling getTask and getStory?

File: lib/ralio.js [hunk 2]
```diff
@@ -340,6 +347,106 @@ Ralio.prototype.point = function (storyID, points, callback) {
...
+  });
+
+};
+
+Ralio.prototype.getStory = function (storyID, callback){
```
Comment (oesmith) [inline lib/ralio.js]: Doesn't Ralio.prototype.story already have the same functionality?

File: lib/ralio.js [hunk 3]
```diff
@@ -340,6 +347,106 @@ Ralio.prototype.point = function (storyID, points, callback) {
   });
 };
 
+Ralio.prototype.task = function (option, projectName, storyID, taskName, callback) {
+  var self = this;
+
+  switch(option){
```
Comment (oesmith) [inline lib/ralio.js]: I'd rather see one function per action - these should be split into a create and delete method.

**Comment by oesmith** [comment]
Hey, thanks for the PR. I've done a very quick code review - I hope my comments all make sense!

Cheers,

Olly

File: lib/ralio.js (hunk 3 above)
Comment (igorescobar) [inline lib/ralio.js]: Sure! no problem! I'll fix it ;)

File: lib/ralio.js (hunk 2 above)
Comment (igorescobar) [inline lib/ralio.js]: Totally true! I didn't see it! You just made I erase two methods! (getStory and getProject) :)

**Comment by igorescobar** [comment]
What about now? :)

**Comment by igorescobar** [comment]
Hey I added a lot of new tests, now I think it's awesome! :)
It was my first time using mocha! smooooooooth! :)

**Comment by igorescobar** [comment]
Did I heard the crowd screaming "nnewwwwww verrrsionn!! new versioonn!!! newww versionn!!" ? :+1:

**Comment by igorescobar** [comment]
Hello? :P

**Comment by oesmith** [comment]
Looks good to me :+1:

It looks like you're pretty familiar with ralio now, so I've added you to the repo. Feel free to push any code you're confident in. If there's something you're not too sure about, please open a pull request and we can have a discussion about it.

Cheers!

Olly

**Comment by igorescobar** [comment]
I tried to push small changes on README.md 
`git push author master
ERROR: Permission to oesmith/ralio.git denied to igorescobar.
fatal: The remote end hung up unexpectedly`

**Comment by oesmith** [comment]
Try again now -- I initially added @igoresc instead of @igorescobar to the repo. It should be right now!

**Pull request merged**